pkg/common/basic.go

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package common
+
+import "strconv"
+
+const theRadixUsedByHumansInTheVastMajorityOfCultures = 10
+
+func UintToString(u uint) string {
+	return strconv.FormatUint(uint64(u), theRadixUsedByHumansInTheVastMajorityOfCultures)
```

Comment:
```suggestion
	return fmt.Sprintf("%d", u)
```
Do we really need a dedicated function for this?

Reply:
@haim-kermany

Reply:
sprintf() is not the proper way to convert int to strings.
every languish has it dedicated method for doing it.
so I try to use the right golang func for it, which is strconv.FormatUint()

however, as usual, golang +lint make it too complex. I must make a conversion to uint64, and can not use 10 as a number.
since it is the right way to do it, but still too complex, it need a dedicated function for it